spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,32 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before(:each) do
+      @user = User.new(name: 'Mesi', posts_counter: 5)
+      @user.save
+      get '/users/1/posts'
+    end
+    it 'checks the correct response status, template and placeholder text' do
+      expect(response).to have_http_status(200)
+      expect(response).to render_template('index')
+      expect(response.body).to include('for all posts')
+    end
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to separate these assertions into individual it blocks.